app/src/main/res/layout/fragment_goals.xml

```diff
@@ -1,115 +1,122 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout
-    xmlns:android="http://schemas.android.com/apk/res/android" 
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:orientation="vertical">
-    
-    <ImageView
-        android:layout_width="100dp"
-        android:layout_height="100dp"
-        android:layout_gravity="center_horizontal"/>
 
     <!-- TODO : Refactor in a goalDisplay layout -->
```

Comment:
We can remove these comments ?

Reply:
I put it to refactor the list of goals but we can remove them, it's not really a priority right now